mantis-common/build.gradle

```diff
@@ -24,11 +24,11 @@ ext {
 
 dependencies {
     api project(":mantis-common-serde")
-    api "io.netty:netty-codec-http:$nettyVersion"
-    api "io.netty:netty-buffer:$nettyVersion"
+    api libraries.nettyCodec
+    api libraries.nettyBuffer
     api group: 'io.netty', name: 'netty-transport-native-epoll', classifier: 'linux-x86_64', version: nettyVersion
```

Comment:
would recommend moving this to libraries.nettyEpoll as well even if it's a little tricky.
It would look like this - `io.netty:netty-transport-native-epoll:4.1.17.Final:linux-x86_64`